docs/index.adoc

```diff
@@ -150,31 +150,53 @@ It *must be unique* for your application.
 
 NOTE: See <<Feature Permissions>> for details on the various states a feature can have.
 
-=== Split targeting strategies and targeting rules
+=== Strategy Splits and general targeting rules
 
-Split targeting provides an ability to rollout features to a limited audience based on targeting rules, for example if you want to turn your feature on 
-for a user only on a mobile device or only with a specific email or userId. Apart from the targeting rules you can also split by percentage, or mix attributes with percentages.
+We refer to a feature flag in an as a Rollout Strategy. Each Rollout Strategy can have _multiple_
+split targeting strategies, i.e. a feature flag in each environment has one default value and zero or more splits.
 
-*How many split targeting strategies can a feature flag have?*
+A split consists of one or more rules. That rule can consist of any combination of matching criteria (such as if you want 
+to turn your feature on for a user only on a mobile device or only with a specific email or 
+userIds). 
 
-There can be *multiple split targeting strategies* assigned to rollout a feature flag. Imagine you have a feature flag of type string which controlls a "submit button color" that can be in multiple states, e.g green, blue, red etc. With rollout strategies, you can serve a `green` value to users on `iOS` devices, `blue` value to users whose emails ending with `gmail.com` and `red` value to users who are in `New Zealand or United States or United Kingdom`.   
+==== Each rule is an AND condition
 
-**How many rules can a strategy have?**
+Each additional rule is always applied as an AND condition - the user is using a mobile device that is iOS *AND*
+their country is Australia.
 
-A split targeting strategy can have multiple rules setup at the same time and as many custom rules as you like.  
+==== Percentage Splits runs
 
-**How many values can be used to set up a rule? **
+There is a special rule which allows you to target based on percentage as well, which is by default done against
+a single field value provided by the client. By default this is sessionId, if that isn't provided userKey is used, 
+but in all cases you can provide a special field to base your percentage rollouts on.
 
-You can set a single value for a rule or you can send an array of values. For example, you can add multiple User Keys to match on, e.g. all users which emails ending with `gmail.com` and all users which emails ending with `my-company.com`
+Percentage splits can be on their own or they can be applied in conjuction with general targeting rules (i.e. field
+based match rules). More detail of Percentage splits is available below under <<Percentage rollout rules>>.
 
+==== Maximum rules
 
-**What split targeting rules types can I use?**
+Each split has zero or more rules associated with it. If it has zero rules, it must at least have a percentage
```

Comment:
never trigger sounds a bit "scary" as it's not clear what exactly wouldn't trigger. I would rephrase "If it has zero rules the strategy will be ignored."